test/e2e-tests.sh

```diff
@@ -135,6 +135,9 @@ toggle_feature responsive-revision-gc Disabled
 # simply cannot pass.
 go_test_e2e -timeout=20m -parallel=300 ./test/scale || failed=1
 
+# Run HPA tests
+go test -timeout=15m -tags=hpa ./test/e2e
```

Comment:
Why not use `go_test_e2e`? Seems like the `-tags` flag overrides it's internal assumption?

Reply:
https://www.digitalocean.com/community/tutorials/customizing-go-binaries-with-build-tags implies that build tags on multiple lines represent `AND` (on the same line it'd be `OR`)

Reply:
Yeah I know ok

Reply:
go_test_2e2 uses user tags if provided also `go test` locally seems not to respect the passed tag list.
```
# Run the given E2E tests. Assume tests are tagged e2e, unless `-tags=XXX` is passed.
# Parameters: $1..$n - any go test flags, then directories containing the tests to run.
function go_test_e2e() {
  local go_test_args=()
  [[ ! " $*" == *" -tags="* ]] && go_test_args+=("-tags=e2e")
...
}```